app/controllers/concerns/sufia/works_controller_behavior.rb

```diff
@@ -17,6 +17,12 @@ def new
       super
     end
 
+    def edit
+      work = GenericWork.find(params[:id])
+      throw "Cannot edit a work that still is being processed" if work.processing?
```

Comment:
@hectorcorrea @grosscol @jcoyne Should this be `raise` instead of `throw`? I don't see a `catch` block around it. Is there a test that makes sure `"Cannot edit a work that still is being processed"` is ever used?